knowledge_repo/converters/ipynb.py

```diff
@@ -1,11 +1,23 @@
 import os
+import nbconvert
```

Comment:
This was an accident on my part, sorry. I've removed this import and fixed-up the relevant commit.